src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py

```diff
@@ -135,9 +137,6 @@ def assert_has_stats(self, output, testcase_path):
 
     self.assertTrue(os.path.exists(testcase_path + '.stats2'))
 
-  def assert_corpus_loaded(self, output, directory):
```

Comment:
They fail on targets compiled with a new clang. I didn't exactly determine why though. @mmoroz may have some ideas because the "corpus loaded" string seems to only be used with dictionary analysis.
I'm searching for the strings in git and the only commits that seem to have this string are from before the compiler-rt move, in other words ancient.